patches/server/0975-Folia-scheduler-and-owned-region-API.patch

```diff
@@ -148,6 +152,7 @@ index 0000000000000000000000000000000000000000..62484ebf4550b05182f693a3180bbac5
 +            if (this.tickCount == RETIRED_TICK_COUNT) {
 +                return false;
 +            }
++            this.server.entitiesWithScheduledTasks.add(entity);
```

Comment:
Inappropriate formatting, when accessing field variables you must use the `this` qualifier to clearly indicate the access is for a field